laws/src/main/scala/cats/laws/discipline/ApplicativeTests.scala

```diff
@@ -30,7 +30,7 @@ import org.scalacheck.{Arbitrary, Cogen, Gen}
 trait ApplicativeTests[F[_]] extends ApplyTests[F] {
   def laws: ApplicativeLaws[F]
 
-  private def makeEqFUnit[A](a: A)(implicit EqFA: Eq[F[A]]): Eq[F[Unit]] =
+  protected[discipline] def makeEqFUnit[A](a: A)(implicit EqFA: Eq[F[A]]): Eq[F[Unit]] =
```

Comment:
So by changing it from `private` to `protected[discipline]` you're making it accessible not only for sub-classes but also for everyone in `discipline` – is that correct?

Reply:
Oh, hm. Maybe that should be `private[discipline]`. The goal is to have it available for us, but not public.

Reply:
```suggestion
  private[discipline] def makeEqFUnit[A](a: A)(implicit EqFA: Eq[F[A]]): Eq[F[Unit]] =
```

Reply:
Although I have no objections in regards to the PR, but I'm sort of curious about the initial implementation there:
```
def makeEqFUnit[A](a: A)(implicit EqFA: Eq[F[A]]): Eq[F[Unit]] =
    Eq.by(fa => laws.F.as(fa, a))
```
What could be a rationale of making `Eq[F[Unit]]` from an arbitrary `a: A` value?
Perhaps, I'm missing something, but it seems like for two different `a` it will produce two `F[Unit]` (even for the same `F` type) that would be considered as unequal to each other. Is that correct?

Reply:
> What could be a rationale of making `Eq[F[Unit]]` from an arbitrary `a: A` value?

We would normally request an `Eq[F[Unit]]` as an implicit parameter, but we cannot do so without breaking binary-compatibility.

> Perhaps, I'm missing something, but it seems like for two different `a` it will produce two `F[Unit]` (even for the same `F` type) that would be considered as unequal to each other. Is that correct?

That is correct, this is just a workaround when you have a constant `a`.